docs/sources/explore/_index.md

```diff
@@ -10,159 +10,53 @@ labels:
-
-When using Explore, the URL in the browser address bar updates as you make changes to the queries. You can share or bookmark this URL.
-
-{{% admonition type="note" %}}
-Explore may generate relatively long URLs, some tools, like messaging or videoconferencing apps, may truncate messages to a fixed length. In such cases Explore will display a warning message and load a default state. If you encounter issues when sharing Explore links in such apps, you can generate shortened links. See [Share shortened link](#share-shortened-link) for more information.
-{{% /admonition %}}
-
-### Generating Explore URLs from external tools
-
-Because Explore URLs have a defined structure, you can build a URL from external tools and open it in Grafana. The URL structure is:
-
-```
-http://<grafana_url>/explore?panes=<panes>&schemaVersion=<schema_version>&orgId=<org_id>
-```
-
-where:
-
-- `org_id` is the organization ID
-- `schema_version` is the schema version (should be set to the latest version which is `1`)
-- `panes` is a url-encoded JSON object of panes, where each key is the pane ID and each value is an object matching the following schema:
-
-```
-{
-  datasource: string; // the pane's root datasource UID, or `-- Mixed --` for mixed datasources
-  queries: {
-    refId: string; // an alphanumeric identifier for this query, must be unique within the pane, i.e. "A", "B", "C", etc.
-    datasource: {
-      uid: string; // the query's datasource UID ie: "AD7864H6422"
-      type: string; // the query's datasource type-id, i.e: "loki"
-    }
-    // ... any other datasource-specific query parameters
-  }[]; // array of queries for this pane
-  range: {
-    from: string; // the start time, in milliseconds since epoch
-    to: string; // the end time, in milliseconds since epoch
-  }
-}
-```
-
-{{% admonition type="note" %}}
-The `from` and `to` also accept relative ranges defined in [Time units and relative ranges]({{< relref "../dashboards/use-dashboards/#time-units-and-relative-ranges" >}}).
-{{% /admonition %}}
-
-## Share shortened link
-
-{{% admonition type="note" %}}
-Available in Grafana 7.3 and later versions.
-{{% /admonition %}}
-
-The Share shortened link capability allows you to create smaller and simpler URLs of the format /goto/:uid instead of using longer URLs with query parameters. To create a shortened link to the executed query, click the **Share** option in the Explore toolbar.
+---
 
-A shortened link that is not accessed will automatically get deleted after a [configurable period](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#short_links) (defaulting to seven days). If a link is used at least once, it won't be deleted.
+## Overview
 
-### Sharing shortened links with absolute time
+Explore is your starting point for querying, analyzing, and aggregating data in Grafana. You can quickly begin creating queries to start analyzing data without having to create a dashboard or visualization.
```

Comment:
"without having to create a dashboard or customize a visualization" maybe?

Reply:
changed the wording